backend/src/main.ts

```diff
@@ -15,7 +15,7 @@ async function bootstrap() {
   app.useGlobalInterceptors(new ClassSerializerInterceptor(app.get(Reflector)));
   app.use(cookieParser());
   app.use(helmet());
-  app.use(csurf({ cookie: true }));
+  // app.use(csurf({ cookie: true }));
```

Comment:
CSRF to be implemented in [TEMPL-34](https://princhcanal.atlassian.net/jira/software/c/projects/TEMPL/boards/5/backlog?view=detail&selectedIssue=TEMPL-34&issueLimit=100)